packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -554,17 +554,39 @@ export namespace EmrCreateCluster {
     TERMINATE_CLUSTER = 'TERMINATE_CLUSTER',
   }
 
+  /**
+   * Spot Allocation Strategies
+   * 
+   * Specifies the strategy to use in launching Spot Instance fleets. Currently, the only option is capacity-optimized (the default), which launches instances from Spot Instance pools with optimal capacity for the number of instances that are launching.
+   *
+   * @see https://docs.aws.amazon.com/emr/latest/APIReference/API_SpotProvisioningSpecification.html
+   * 
+   */
+  export enum SpotAllocationStrategy {
+    /**
+     * capacity-optimized, which launches instances from Spot Instance pools with optimal capacity for the number of instances that are launching.
+     */
+    CAPACITY_OPTIMIZED = 'capacity-optimized',
+  }
+
   /**
    * The launch specification for Spot instances in the instance fleet, which determines the defined duration and provisioning timeout behavior.
    *
    * @see https://docs.aws.amazon.com/emr/latest/APIReference/API_SpotProvisioningSpecification.html
    *
    */
   export interface SpotProvisioningSpecificationProperty {
+    /**
+     * Specifies the strategy to use in launching Spot Instance fleets. Currently, the only option is capacity-optimized (the default), 
+     * which launches instances from Spot Instance pools with optimal capacity for the number of instances that are launching.
+     *
+     * @default - No allocation strategy
```

Comment:
Can you add a touch more colour about what no strategy means? I'm not that familiar with EMR so forgive me for not providing a suggestion

Reply:
Done